ui/src/ducks/app/monitoring.js

```diff
@@ -20,17 +22,18 @@ const defaultState = {
     list: [],
     error: null,
     isLoading: false,
-    isRefreshing: false
+    isRefreshing: false,
   },
   cluster: {
     apiServerStatus: 0,
     kubeSchedulerStatus: 0,
     kubeControllerManagerStatus: 0,
     error: null,
+    prometheusPodStatusMessage: null,
```

Comment:
IMHO, this should not be in `monitoring.cluster`, simply `monitoring`, or maybe even go as far as:

```
const defaultState = {
  // ...
  cluster: { ... },
  prometheus: {
    isAvailable: false, // the old `isPrometheusApiUp`
    podStatusMessage: null,
  },
}
```

That could help keep things organized. But feel free to put this in a debt ticket.

Reply:
> IMHO, this should not be in `monitoring.cluster`, simply `monitoring`, or maybe even go as far as:
> 
> ```
> const defaultState = {
>   // ...
>   cluster: { ... },
>   prometheus: {
>     isAvailable: false, // the old `isPrometheusApiUp`
>     podStatusMessage: null,
>   },
> }
> ```
> 
> That could help keep things organized. But feel free to put this in a debt ticket.

Will use`isPrometheusVolumeProvisioned` prop in cluster to control the banner visibility.